### PR TITLE
[LinearSolversApplication][Doc] Update MKL doc for Ubuntu

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Update MKL doc for Ubuntu for ONEAPI 2022

**🆕 Changelog**

- Update MKL doc for Ubuntu for ONEAPI 2022

### CODE REVIEW

**Comment by loumalouomega** [comment]
@roigcarlo still some CI are triggered even if I modify a markdown file

File: applications/LinearSolversApplication/README.md [hunk 1]
```diff
@@ -168,36 +168,38 @@ In case you have installed [MKL](https://software.intel.com/en-us/mkl) (see belo
...
+# use wget to fetch the Intel repository public key
+wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+# add to your apt sources keyring so that archives signed with this key will be trusted.
+sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+# remove the public key
+rm GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+# Configure apt client to use Intel repository
+sudo add-apt-repository "deb https://apt.repos.intel.com/oneapi all main"
+# Install all HPC related dependencies
+sudo apt install intel-hpckit
```
Comment (philbucher) [inline applications/LinearSolversApplication/README.md]: this is much larger than mkl, like several GB!

Can you please add a separate section instead?

**Review by philbucher — CHANGES_REQUESTED** [review]

**Comment by philbucher** [comment]
> @roigcarlo still some CI are triggered even if I modify a markdown file

hmm I am also curious why this happens

File: applications/LinearSolversApplication/README.md (hunk 1 above)
Comment (loumalouomega) [reply applications/LinearSolversApplication/README.md]: I can only put the MKL one...

Comment (loumalouomega) [reply applications/LinearSolversApplication/README.md]: Happy?

**Comment by roigcarlo** [comment]
Mm I think this is not the problem. Checks are not being triggered, as intended, but github is still enforcing you to pass the mandatory checks, which is different. 

I am investigating

**Comment by roigcarlo** [comment]
Found it. I forgot to implement this: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks

**Comment by roigcarlo** [comment]
@loumalouomega You should be able to merge master and trigger the CI now (hopefully)

**Comment by loumalouomega** [comment]
> @loumalouomega You should be able to merge master and trigger the CI now (hopefully)

Let's see

**Review by philbucher — APPROVED** [review]

**Pull request merged**